index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Kindly try to fix your page styles, so it looks appropriate

- Your page now:
![Screenshot_20220630_062647](https://user-images.githubusercontent.com/25227022/176599677-cf073121-5c23-484f-8975-8d25b7bbb261.png)